resources/views/comments/delete.blade.php

```diff
@@ -0,0 +1,26 @@
+@extends('master')
+
+@section('title')
+Opmerking verwijderen
+@endsection
+
+@section('content')
+
+<h1>Opmerking verwijderen</h1>
+
+<hr />
+
+@include ('errors.list')
+
+{!! Form::open(['url' => 'events/'.$comment->id, 'method' => 'DELETE']) !!}
+
+<p>Weet je zeker dat je comment bij {{$comment->entityDescription}} wilt verwijderen?</p>
```

Comment:
Je gebruikt nu de termen 'opmerking' en 'comment' door elkaar heen in de front-end. Het lijkt me goed om één term aan te houden. Mijn voorstel is 'opmerking' (gewoon lekker in het Nederlands houden waar kan).